pytests/test_microgrid.py

```diff
@@ -3,22 +3,23 @@
 
 """Tests for the frequenz.api.microgrid package."""
 
-# pylint: disable=import-outside-toplevel
```

Comment:
Do you know why did you need to move this to every function? Or is it because it is like that in the repo-config template? If that is the case, I think we should update the repo-config template instead.